File: src/main/webapp/directive/employee/employeeController.js
(function (angular) {
    'use strict';
    angular.module('gymApp').controller('EmployeeController', employeeController);

    employeeController.$inject = ['$scope', 'EmployeeService'];

    function employeeController($scope, EmployeeService) {
        $scope.findAllEmployees = function () {
            EmployeeService.findAll().then(function (employees) {
                $scope.employees = employees;
            });
        };

        $scope.findAllEmployees();
    }
})(angular);

File: src/main/java/com/tylersadler/controller/EmployeeController.java
package com.tylersadler.controller;

import com.tylersadler.dto.EmployeeDto;
import com.tylersadler.service.EmployeeService;
import org.springframework.http.MediaType;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RequestMethod;
import org.springframework.web.bind.annotation.RestController;

import javax.inject.Inject;
import java.util.List;


@RestController
@RequestMapping("/api")
public class EmployeeController {

    private final EmployeeService employeeService;

    @Inject
    public EmployeeController(EmployeeService employeeService){
        this.employeeService = employeeService;
    }

    @RequestMapping(value = "/employees", method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE)
    public List<EmployeeDto> findAll(){
        return employeeService.findAll();
    }
}